dixTable.py

```diff
@@ -229,40 +228,25 @@ def updateOrCreatePages(location, pairName, updateStats, editToken):
     languages = set(comprehensiveList(args.languages))
     logging.info('Using languages {}'.format(languages))
 
-    dirs = [
-        ('incubator', r'<name>(apertium-(\w{2,3})-(\w{2,3}))</name>'),
-        ('nursery', r'<name>(apertium-(\w{2,3})-(\w{2,3}))</name>'),
-        ('staging', r'<name>(apertium-(\w{2,3})-(\w{2,3}))</name>'),
-        ('trunk', r'<name>(apertium-(\w{2,3})-(\w{2,3}))</name>'),
+    dirPaths = [
+        'incubator',
+        'nursery',
+        'staging',
+        'trunk',
     ]
 
-    for (dirPath, dirRegex) in dirs:
-        svnData = str(subprocess.check_output('svn list --xml https://svn.code.sf.net/p/apertium/svn/%s/' % dirPath, stderr=subprocess.STDOUT, shell=True), 'utf-8')
-        for langDir in re.findall(dirRegex, svnData, re.DOTALL):
-            if set(langDir[1:]) <= languages:
-                primaryPairs[frozenset({langDir[1], langDir[2]})] = (dirPath, langDir[0])
-            elif langDir[1] in languages or langDir[2] in languages:
-                secondaryPairs[frozenset({langDir[1], langDir[2]})] = (dirPath, langDir[0])
-
-    svnData = str(subprocess.check_output('svn list --xml https://svn.code.sf.net/p/apertium/svn/incubator/', stderr=subprocess.STDOUT, shell=True), 'utf-8')
-    for dixFileElem in etree.fromstring(svnData).findall('.//name'):
-        dixFileName = dixFileElem.text
-        if re.match(r'apertium-[^\.]+\.[^\.]+\.dix(?:\.xml)?', dixFileName):
-            try:
-                pair = dixFileName.split('.')[1].split('-')
-                dixLink = 'https://svn.code.sf.net/p/apertium/svn/incubator/%s' % dixFileName
-                if len(pair) > 1 and pair[0] in languages or pair[1] in languages:
-                    if frozenset(pair) not in primaryPairs and frozenset(pair) not in secondaryPairs:
-                        if pair[0] in languages and pair[1] in languages:
-                            primaryPairs[frozenset(pair)] = ('incubator', dixFileName.split('.')[0])
-                        else:
-                            secondaryPairs[frozenset(pair)] = ('incubator', dixFileName.split('.')[0])
-                    else:
-                        logging.error('%s already recorded! Skipping!' % str(pair))
-            except IndexError:
-                pass
-
-    #print(primaryPairs, secondaryPairs)
+    gitData = json.loads(subprocess.check_output('curl -X GET https://apertium.projectjj.com/stats-service/packages/ -H accept: application/json | python -m json.tool', shell=True))["packages"]
+    for package in gitData:
+        langDir = re.findall(r"(apertium-(\w+)-(\w+))", package["name"], re.DOTALL)
+        if langDir:
+            langDir = langDir[0]
+            for topic in package["topics"]:
```

Comment:
Seems like you can just do an intersection here between `package["topics"]` and `["apertium-{}".format(dir) for dir in dirPaths]`